server/templates/controller.html

```diff
@@ -1,66 +1,85 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <script src="/static/vendor/jquery.min.js" charset="UTF-8" type="text/javascript"></script>
-    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
-    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
-    <script src="/static/js/controller.js" charset="UTF-8" type="text/javascript"></script>
-    <style>
-      body {
-        padding: 10;
-        margin: 10;
-      }
-
-      .form-field {
-        margin-bottom: 25px;
-      }
-      .data-field {
-        display: block;
-        margin-bottom: 15px;
-      }
-
-      #tag-holder label {
-        display: inline-block;
-        margin-bottom: 10px;
-      }
-    </style>
-  </head>
-
-  <body>
-    <p>Buttons and things.</p>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Fb - Add Bob</title>
+
+  <!-- load font -->
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css" />
+
+  <!-- load MUI -->
+  <link href="//cdn.muicss.com/mui-0.9.16/css/mui.min.css" rel="stylesheet" type="text/css" />
+  <script src="//cdn.muicss.com/mui-0.9.16/js/mui.min.js"></script>
+
+  <!-- load CSS, overrides MUI -->
+  <link rel="stylesheet" type="text/css" href="/static/css/controller.css">
+
+  <!-- load js -->
+  <script src="/static/vendor/socket.io.js" charset="UTF-8" type="text/javascript"></script>
+  <script src="/static/vendor/jquery.min.js" charset="UTF-8" type="text/javascript"></script>
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
+  <script src="/static/js/controller.js" charset="UTF-8" type="text/javascript"></script>
+</head>
+
+<body>
+
+  <div class="mui-container">
+    <div id="navbar" class="mui-appbar">
+      <table width="100%">
+        <tr style="vertical-align:middle;">
```

Comment:
We ought to avoid inline styles so that it's more clear where styling is being applied (in CSS files)